### PR TITLE
test models

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR tests the Leave one Channel out (LOCO) analysis elastic net models.

### CODE REVIEW

File: 11.bulk_Morphology_Elastic_Network_LOCO/2.test_models/scripts/1.test_regression_multi_output.py [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+        "neg_mean_absolute_error",
+        "neg_mean_squared_error",
+        "well",
+        "treatment",
+        "r2",
+        "log10_neg_mean_absolute_error",
+        "log10_neg_mean_squared_error",
+        "log10_explained_variance",
+    ]
+)
```
Comment (axiomcura) [inline 11.bulk_Morphology_Elastic_Network_LOCO/2.test_models/scripts/1.test_regression_multi_output.py:280]: I am trying to understand why you are dropping all these columns, what is `var_df` trying to represent here and how are you using it. Please consider adding some comments to justify `var_df`.

File: 11.bulk_Morphology_Elastic_Network_LOCO/2.test_models/scripts/1.test_regression_multi_output.py [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+var_df = pd.merge(
+    var_df,
+    results_df.groupby(
+        ["cytokine", "data_split", "shuffle", "channel_feature_combinations_key"]
+    ).r2.unique(),
+    left_index=True,
+    right_index=True,
+)
+var_df.reset_index(inplace=True)
+var_df.head()
```
Comment (axiomcura) [inline 11.bulk_Morphology_Elastic_Network_LOCO/2.test_models/scripts/1.test_regression_multi_output.py:294]: Ah, now I see what you're doing. But the question remains whether you should add comments to help other readers to understand.

**Review by axiomcura — APPROVED** [review]
LGTM, left few comments. Nice work!

File: 11.bulk_Morphology_Elastic_Network_LOCO/2.test_models/scripts/1.test_regression_multi_output.py [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+df = pd.DataFrame.from_dict(output_metric_scores)
+df["r2"] = r2
+df["cytokine"] = cytokine
+df["data_split"] = data_split
+df["shuffle"] = shuffle
+df["predicted_value"] = predictions
+df["actual_value"] = y_selected
+df["log10_neg_mean_absolute_error"] = -np.log10(-df["neg_mean_absolute_error"])
+df["log10_neg_mean_squared_error"] = -np.log10(-df["neg_mean_squared_error"])
+df["log10_explained_variance"] = -np.log10(df["explained_variance"])
```
[the remark this answers is not in the document]
Comment (MikeLippincott) [reply 11.bulk_Morphology_Elastic_Network_LOCO/2.test_models/scripts/1.test_regression_multi_output.py:248]: addressed

File: 11.bulk_Morphology_Elastic_Network_LOCO/2.test_models/scripts/1.test_regression_multi_output.py (hunk 1 above)
Comment (MikeLippincott) [reply 11.bulk_Morphology_Elastic_Network_LOCO/2.test_models/scripts/1.test_regression_multi_output.py:280]: added in more comments

**Pull request merged**